src/xenia/emulator.cc

```diff
@@ -307,14 +313,9 @@ const std::unique_ptr<vfs::Device> Emulator::CreateVfsDeviceBasedOnPath(
   } else if (extension == ".7z" || extension == ".zip" || extension == ".rar") {
     xe::ShowSimpleMessageBox(
         xe::SimpleMessageBoxType::Error,
-        fmt::format("Xenia does not support running {} files, the file you "
-                    "actually want to run is inside of this {} file.",
-                    extension, extension));
-    // user likely compressed the file themselves, then suffered a traumatic
-    // brain injury that caused them to forget how archives work
-
-    LaunchWebBrowser(fmt::format(
-        "https://www.google.com/search?q=how+to+extract+{}+file", extension));
+        fmt::format("Xenia does not support running {} files.", extension));
+    // let's not open the webpage without user consent
+    // LaunchWebBrowser(fmt::format("https://www.google.com/search?q=how+to+extract+{}+file", extension));
 
     xe::FatalError("Terminating due to user's lack of basic computer skills.");
```

Comment:
It might be good to remove (or at least change) this message to remove this very hostile message as well, since it very much questions the user's intelligence.